Source/Core/Core/ConfigManager.cpp

```diff
@@ -96,6 +96,7 @@ void SConfig::SaveSettings()
   ini.Save(File::GetUserPath(F_DOLPHINCONFIG_IDX));
 
   Config::Save();
+  Config::Load();  // Stupid workaround because various UI things don't set onion config
```

Comment:
As in they just haven't been moved to work with the config, or they deliberately don't set it? If it's the former, then... maybe fix that first first before this feature set and leaving a kludge in.

Reply:
It is the former, but that is a major rewrite that is still waiting to happen.

Reply:
Well, I guess I could go to every option in the UI that's relevant to this issue, and have it set onion config in addition to SConfig. Would that work for you?